src/emitter.c

```diff
@@ -1977,6 +1980,10 @@ yaml_emitter_write_plain_scalar(yaml_emitter_t *emitter,
 
     emitter->whitespace = 0;
     emitter->indention = 0;
+    if (emitter->root_context)
+    {
+        emitter->open_ended = 1;
+    }
```

Comment:
See above comment. This is basically a revert from a previous commit, but the test suite doesn't catch that.

Reply:
I pushed this change up.